readme.md

```diff
@@ -433,6 +433,7 @@
 - [Vehicle Security and Car Hacking](https://github.com/jaredthecoder/awesome-vehicle-security#readme)
 - [Web Security](https://github.com/qazbnm456/awesome-web-security#readme) - Security of web apps & services.
 - [Lockpicking](https://github.com/meitar/awesome-lockpicking#readme) - The art of unlocking a lock by manipulating its components without the key.
+- [Pentest](https://github.com/enaqx/awesome-pentest#readme) - Penetration testing resources and tools.
```

Comment:
`Pentest` => `Penetration Testing` since that's the title of your list

Reply:
> The entry in the Awesome list should:
Include a short description about the project/theme of the list. **It should not describe the list itself.**
Example: - [Fish](…) - User-friendly shell., not - [Fish](…) - Resources for Fish..